frontend/src/components/Comments/CommentSpacer.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div class="bar"><slot/></div>
```

Comment:
new line for the slot would be good just so it's easy to see what's going on.

Reply:
donezo